lib/puppetserver/ca/certificate_authority.rb

```diff
@@ -50,7 +50,7 @@ def revoke_certs(certnames)
                     body: REVOKE_BODY,
                     type: :revoke)
 
-        results.reduce {|prev, curr| worst_result(prev, curr) }
+        results.reduce { |prev, curr| worst_result(prev, curr) }
```

Comment:
What's with this?

Reply:
Asymmetrical spacing. Style guides I found ([ex](https://github.com/github/rubocop-github/blob/master/STYLEGUIDE.md)) all say spaces after curly braces, which I think is generally what we do elsewhere.

Reply:
Huh, I don't think I've noticed that lately.